### ISSUE TITLE
Trained without data augmentation?

### ISSUE THREAD

**Issue by nandi-zhang** [benjs/nfnets_pytorch#2]
Thanks for the great work on the pytorch implementation of NFNet! The accuracies achieved by this implementation are pretty impressive also and I am wondering if these training results were simply derived from the training script, that is, without data augmentation.

**Comment by benjs** [comment]
The results were derived from the pretrained weights, provided by the authors in their original repo :)

As the data augmentation is not implemented yet, the training script should reach the baseline accuracy as provided in the paper (F0: 80%). This, however, is not tested yet. I am currently adding mutli gpu support (see branch distributed) and try to reach the baseline accuracy on AWS. This will unfortunatly take some time as I am writing exams the next weeks :/

**Comment by nandi-zhang** [comment]
Got it. Just take your time and good luck on the exams. :))
